### ISSUE TITLE
[BUG] Headings must not be empty with images

### ISSUE THREAD

**Issue by tiagoporto** [yeonjuan/html-eslint#452]
**Describe the bug**
@html-eslint/no-empty-headings returns an error when used with images.

**To Reproduce**

```html
<h1><img src="./img.svg" alt="Images description" /></h1>
```
**Expected behavior**
I expect no errors because it is a valid HTML, and the image text should not be ignored.

**Comment by yeonjuan** [comment]
Hi @tiagoporto Thank you for your report. 👍  This is a bug.
